db/seeds.rb

```diff
@@ -6,15 +6,49 @@
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
 
+
+# first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+# second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+
+# (1..4).each do |i|
+#     Post.create(author: first_user, title: "Hello ##{i}", text: "This is my post #{i}")
+# end
+
+# first_post = Post.first
+
+# (1..6).each do |i|
+#     Comment.create(post: first_post, author: second_user, text: "comment #{i}" )
+# end
+
 first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
 second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+User.create(name: 'User 3', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
```

Comment:
It is great implementation for testing. I love the idea you put in here. 

Based on your great effort I would suggest you something to learn. Kindly check for Swagger for api documentation. You will need it in future works. For example in final capstone and hopefully in your new job as Full Stack developer. Learning Swagger documentation to present your code in backend it such a great idea. If you need anything ping me on linked in. You can find my linked in on my Github profile!.

Reply:
Thank you so much!! I'll chat you up on LinkedIn during the weekend